docs/conf.py

```diff
@@ -0,0 +1,40 @@
+# Configuration file for the Sphinx documentation builder.
+
+# -- Project information
+
+project = 'Thunderbird Websites'
+copyright = '2024, Mozilla'
```

Comment:
Not MZLA Technologies Corporation?

Reply:
I can open a related ticket in the atn repo later